### ISSUE TITLE
Error message "Cannot move notebook to this location" or "To manually sort the notes..." multiple times a day when clicking

### ISSUE THREAD

**Issue by beatrixwillius** [laurent22/joplin#7720]
Because of arthritis in my hands I use a graphics tablet instead of a mouse - always. The tablet seems to be too sensitive for Joplin. Multiple times a day when I click something I get messages like the ones below:
![c6cfb4a38f2888b21c55251efbaacf080b22d8ab](https://user-images.githubusercontent.com/1361928/216764594-aee95f6c-2183-448c-abbd-e4c439a56397.png)
![3e6a2657cade3e6f7b6f6a824074194753e53c90](https://user-images.githubusercontent.com/1361928/216764676-00757a99-8ee3-4b56-8672-3b7a6ba9dea2.png)

I experimented a bit. The "Cannot move notebook to this location" occurs if I don't keep my pen vertical. I uploaded a YouTube video to https://youtu.be/BtJhI3MBcew .

I haven't seen anything like this in other apps.

**Comment by dpoulton-github** [comment]
[Linked forum thread](https://discourse.joplinapp.org/t/using-joplin-with-a-graphics-tablet/29428) shows OP is using Joplin v2.9.17 on macOS High Sierra.

This occasionally happens to me when I click on a note or notebook to highlight it but *ever-so-slightly* move my mouse at the same time. This is normally because my mouse is still moving slightly as I left-click. It then triggers the "move / sort" dialogs. I would guess that the minimum movement required to trigger the action is not much and adding a few more pixels to the tolerence (if that can be done) would reduce accidental activations.
*Joplin v2.9.17 Kubuntu 22.04.1*

**Comment by laurent22** [comment]
I had a look and I thought indeed we would just need to increase the move distance, but it seems we're using the built-in drag and drop functionality and it cannot be configured. Perhaps changing the sensitivity of the tablet itself would help?

In the meantime I leave the issue open anyway. Perhaps one solution would be to disable drag and drop completely when it's not needed

**Comment by beatrixwillius** [comment]
The Wacom driver has some options for individual apps. But none for the move distance.
